src/test/kotlin/test/ktortemplate/core/model/TestModels.kt

```diff
@@ -0,0 +1,49 @@
+@file:UseSerializers(InstantAsStringSerializer::class)
```

Comment:
What does this do? It registers the `InstantAsStringSerializer` as the default serializer for instants?

Reply:
Yes, only in the context of this file. It saves trouble for adding `@Serializable(with = InstantAsStringSerializer::class)` for each fields.